### DISCUSSION TITLE
Vite Support

### DISCUSSION THREAD

**Discussion by yoochangoh** [coreui/coreui-free-react-admin-template#417] [Q&A] [answered]
Hello,

I am wondering if there are any plans to support Vite as a module bundler.

Is the template that comes with a CoreUI PRO purchase supporting Vite?

Thank you.

**Comment by mrholek** [comment, marked as answer]
Yes, probably this week we will release vite version

**Reply by yoochangoh** [reply]
Thank you for your response.

I am eagerly anticipating the Vite version update! 🙌

**Reply by mrholek** [reply]
Currently, Vite is the default bundler for our React admin template. The CRA version is still available in the following branch - https://github.com/coreui/coreui-free-react-admin-template/tree/cra
